### ISSUE TITLE
Update umbrel connector video

### ISSUE THREAD

**Issue by bumi** [getAlby/lightning-browser-extension#1008]
### Is your feature request related to a problem? Please describe

Umbrel released version 0.5 and changed the interface. 
We need to update the instructions and the video how to connect Alby to Umbrel.

Details should also be added to the guides here: https://community.getumbrel.com/t/guide-connect-umbrel-to-your-wallet/7653

**Comment by bumi** [comment]
maybe we should remove it when it gets too old? @reneaaron

**Comment by reneaaron** [comment]
Agree, we should remove outdated ones. I can also record a new one if that makes sense?

/cc @MoritzKa

**Comment by JPLACLAU** [comment]
I could do this, But this issue is almost a year old. If this is still an issue I'd love to make that instructions video. Let me know.

**Comment by reneaaron** [comment]
Thanks for offering your help here! :raised_hands: 

We recently replaced the video with a screenshot of where you can find the information. Since UIs change quite frequently I am not sure if it's worth it keeping them up to date. 🤔

Do you think for an average user it's enough to have this screenshot & a guide or would it be better to have some video guide? What do you think @JPLACLAU?

**Comment by JPLACLAU** [comment]
I think if there is already a solution that it already works, for the time being, there is no reason to add extra work for something that is doing its purpose well. Constant UI updates are terrible for video tutorials. 

However, In some time when there is more stability, less ui upgrades and more users... we might reconsider doing some short video tutorials. I believe Screenshots are fine for advanced users. But Videos will always be a plus and some people will always prefer them, specially beginners.

**Comment by stackingsaunter** [comment]
Closing as there's an updated screenshot with step-by-step instructions. It's much easier flow now than before, I think it does not need a video.
